pyright/node.py

```diff
@@ -100,7 +100,7 @@ def _install_node_env() -> None:
 
 def run(
     target: Target, *args: str, **kwargs: Any
-) -> Union['subprocess.CompletedProcess[bytes]', 'subprocess.CompletedProcess[str]']:
```

Comment:
I'm truly happy that this PR doesn't go through, as the X|Y syntax requires Python 3.10.